scene/animation/tween.cpp

```diff
@@ -821,17 +821,20 @@ bool Tween::remove(Object *p_object, String p_key) {
 		call_deferred("remove", p_object, p_key);
 		return true;
 	}
+	List<List<InterpolateData>::Element *> for_removal;
 	for(List<InterpolateData>::Element *E=interpolates.front();E;E=E->next()) {
 
 		InterpolateData& data = E->get();
 		Object *object = ObjectDB::get_instance(data.id);
 		if(object == NULL)
 			continue;
-		if(object == p_object && data.key == p_key) {
-			interpolates.erase(E);
-			return true;
+		if(object == p_object && (data.key == p_key || p_key == "")) {
+			for_removal.push_back(E);
```

Comment:
Would be nice if we still break out of the loop if `p_key != ""`, for performance considerations :smile: